instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -108,6 +108,26 @@
 }
 
 
+class MyHooks(object):
```

Comment:
We are using Python3 only, a class should not inherit from `object`.

Reply:
@ocelotl , will make that changes.
Thanks for the review, appreciate.